exercise03/README.md

```diff
@@ -0,0 +1,3 @@
+## Exercise 3
+
+Given the tables and records defined `setup.sql`.  Write a query that returns all records and a count where `name` is not unique.
```

Comment:
A couple things:

1) "count" reads ambiguously to me, what do we want a count of? the number of times each duplicated name is repeated? the count of duplicate names? the number of records that could be considered duplicate?

2) I'd use a comma instead of a period, "...records defined `setup.sql`, write a query..."